pkg/util/fixtures/populate/populate.go

```diff
@@ -0,0 +1,145 @@
+			N: fromBase10("14314132931241006650998084889274020608918049032671858325988396851334124245188214251956198731333464217832226406088020736932173064754214329009979944037640912127943488972644697423190955557435910767690712778463524983667852819010259499695177313115447116110358524558307947613422897787329221478860907963827160223559690523660574329011927531289655711860504630573766609239332569210831325633840174683944553667352219670930408593321661375473885147973879086994006440025257225431977751512374815915392249179976902953721486040787792801849818254465486633791826766873076617116727073077821584676715609985777563958286637185868165868520557"),
+			E: 3,
+		},
+		D: fromBase10("9542755287494004433998723259516013739278699355114572217325597900889416163458809501304132487555642811888150937392013824621448709836142886006653296025093941418628992648429798282127303704957273845127141852309016655778568546006839666463451542076964744073572349705538631742281931858219480985907271975884773482372966847639853897890615456605598071088189838676728836833012254065983259638538107719766738032720239892094196108713378822882383694456030043492571063441943847195939549773271694647657549658603365629458610273821292232646334717612674519997533901052790334279661754176490593041941863932308687197618671528035670452762731"),
+		Primes: []*big.Int{
+			fromBase10("130903255182996722426771613606077755295583329135067340152947172868415809027537376306193179624298874215608270802054347609836776473930072411958753044562214537013874103802006369634761074377213995983876788718033850153719421695468704276694983032644416930879093914927146648402139231293035971427838068945045019075433"),
+			fromBase10("109348945610485453577574767652527472924289229538286649661240938988020367005475727988253438647560958573506159449538793540472829815903949343191091817779240101054552748665267574271163617694640513549693841337820602726596756351006149518830932261246698766355347898158548465400674856021497190430791824869615170301029"),
+		},
+	}
+	sample2048Key.Precompute()
+}
+
+// Walk is a recursive struct value population function. Given a pointer to an arbitrarily complex value v, it fills
+// in the complete structure of that value, labelling each element with the path by which it was found.
+//
+// This function has the following caveats:
+//  - Signed integers are set to int(3)
+//  - Unsigned integers are set to uint(3)
+//  - Floating point numbers are set to float(3.0)
+//  - Booleans are set to True
+//  - Arrays and slices are allocated 3 elements
+//  - Maps are allocated 3 elements
+//  - Only map[string][string] types are supported
+//  - strings are set to the value of the path taken to reach the string
+func Walk(v interface{}) {
+	val := reflect.ValueOf(v)
+	if val.Kind() != reflect.Ptr {
+		panic("argument is not a pointer to a value")
+	}
+	walk(val, "")
+}
+
+// walk fills in the complete structure of a complex value v using path as the root of the labelling.
+func walk(v reflect.Value, path string) {
```

Comment:
right at the top of this function, do:

```golang
// special cases
switch v.Interface().(type) {
case []byte:
	v.SetBytes([]byte(path))
	return

case *rsa.PrivateKey:
	// use a dummy value because the zero value cannot be marshalled
	v.Set(reflect.ValueOf(dummyKey))
	return

case []api.IdentityProvider:
	// override all the supported provider kinds here
	v.Set(reflect.ValueOf([]api.IdentityProvider{{Provider: &api.AADIdentityProvider{}}}))
}
```

the `case []api.IdentityProvider` is needed to fully walk an api.OpenShiftManagedCluster.